cmd/protoc-gen-cpp-tableau-loader/helper/keyword.go

```diff
@@ -1,8 +1,28 @@
 package helper
 
+import (
+	"unicode"
+
+	"github.com/elliotchance/pie/v2"
+)
+
 var cppKeywords map[string]bool
 
-func KeywordEscape(str string) string {
+func VarNameEscape(str string) string {
+	// Filter invalid runes
```

Comment:
- this is a small plugin, DO NOT import big package "github.com/elliotchance/pie/v2" for only one feature
- rename `VarNameEscape` -> `escapeIdentifier`, refer: https://en.cppreference.com/w/cpp/language/identifiers
```
package main

import (
	"fmt"
	"strings"
	"unicode"
)

func removeNonAlnum(s string) string {
	var result strings.Builder
	for _, r := range s {
		if unicode.IsLetter(r) || unicode.IsNumber(r) || r == '_' {
			result.WriteRune(r)
		}
	}
	return result.String()
}

func main() {
	str := "Hello, World!_123"
	fmt.Println(removeNonAlnum(str)) // Prints: HelloWorld_123
}
```

Reply:
done